src/hermes/commands/cli.py

```diff
@@ -56,6 +58,28 @@ def main() -> None:
     # Actually parse the command line, configure it and execute the selected sub-command.
     args = parser.parse_args()
 
-    args.command.load_settings(args)
-    args.command.patch_settings(args)
-    args.command(args)
+    logger.init_logging()
+    log = logger.getLogger("hermes.cli")
+    log.debug("Running hermes with the following command line arguments: %s", args)
+
+    try:
+        log.info("Loading settings...")
```

Comment:
I don't think there's value in showing this to the user. Of course settings are loaded. That's what every program does. 😅 

Maybe it would be useful to know where the settings are read from. But on the other hand, we can't even run the first stage without a hermes.toml. But in that case, the message is just debug and the user doesn't know if they don't look into the log file.

I would change this to `debug` in this particular case.

Reply:
```suggestion
        log.debug("Loading settings...")
```